precompile/contracts/hubblebibliophile/margin_account.go

```diff
@@ -10,8 +10,8 @@ import (
 )
 
 const (
-	MARGIN_ACCOUNT_GENESIS_ADDRESS        = "0x0300000000000000000000000000000000000070"
-	VAR_MARGIN_MAPPING_STORAGE_SLOT int64 = 11
+	MARGIN_ACCOUNT_GENESIS_ADDRESS        = "0x0300000000000000000000000000000000000001"
+	VAR_MARGIN_MAPPING_STORAGE_SLOT int64 = 10
```

Comment:
Did this slot change on contract side from 11 to 10 ?
could find the change in PR related to this in hubble-protocol. Is this a fix ?

Reply:
It is a fix. Somehow got crept in here https://github.com/hubble-exchange/subnet-evm/pull/52/files#diff-558ab4d135487100d7a0aa2d7dc2794369385b2cad9a346ad53cac299d94995eL14